integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/viewpopenqueueservice/test/ViewPopEnqueueTest.java

```diff
@@ -0,0 +1,120 @@
+package org.wso2.carbon.esb.viewpopenqueueservice.test;
+
+import org.wso2.carbon.automation.extensions.servers.jmsserver.client.JMSQueueMessageProducer;
+import org.wso2.carbon.automation.test.utils.http.client.HttpRequestUtil;
+import org.wso2.carbon.message.processor.stub.MessageProcessorAdminService;
+import org.wso2.carbon.message.store.stub.MessageInfo;
+import org.wso2.esb.integration.common.clients.mediation.MessageStoreAdminClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Tests to verify the functionality
```

Comment:
Doc comments need to end with a persiod. Please refer to https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
Also it's better if we can describe what functionality is tested in the class comment.

Reply:
This test has been removed from the PR and no longer exists. Whatever tests that existed here has been moved to two separate test classes